policies/README.md

```diff
@@ -0,0 +1,64 @@
+# OPA policies
+Policies are rules that can be applied when any of your cluster object is _createdn_, _updated_, _patched_ and _deleted_.
+Policies are always executed _before_ the action takes place and their goal is to allow or prevent the requested modification.
+
+This solution is very fast in term of required computing time thanks to the efficienty of the OPA gatekeeper.
+
+
+## How to use policies
+To use OPA gatekeeper you need to:
+- Deploy a Release using Prebuilt Image. See [link](../infrastructure/policy-enforcement/README.md).
```

Comment:
Not clear to me. What did you want to say? That you need to have a gatekeeper running in your cluster, so you have to depoy it?